atm.js

```diff
@@ -1,3 +1,147 @@
+      var accountsTotal = 0;
+      accountsTotal = savingAccountTotal + checkingAccountTotal;
+      if ( checkingWithdrawValue <= accountsTotal){
+        checkingWithdrawValue = checkingWithdrawValue - checkingAccountTotal;
+        savingAccountTotal = savingAccountTotal - checkingWithdrawValue;
+        savingAccountTotal = String(savingAccountTotal);
+        $("#savings .balance").html("$" + savingAccountTotal);
+        checkingAccountTotal = checkingAccountTotal - checkingAccountTotal;
+        checkingAccountTotal = String(checkingAccountTotal);
+        $("#checking .balance").html("$" + checkingAccountTotal);
+      }
+      else
+      {
+        alert("Sorry, there is not enough in your accounts to withdraw this much");
+      }
+
+    }
+    changeBoxColorChecking();
+    changeBoxColorSaving();
+  });
+
+
+
+  //SAVINGS ACCOUNT
+  //Deposit Button
+  var savingDepositButton = $("#savings .deposit");
+
+
+  //if click on checking account deposit button, then inserts into checking balance
+  savingDepositButton.on("click", function(){
+    var savingInputValue = $("#savings .input");
+    savingInputValue = parseInt(savingInputValue.val());
+    savingAccountTotal = parseInt(savingAccountTotal);
+    savingAccountTotal = savingAccountTotal + savingInputValue;
+    savingAccountTotal = String(savingAccountTotal);
+    $("#savings .balance").html("$" + savingAccountTotal);
+    changeBoxColorChecking();
+    changeBoxColorSaving();
+  });
+
+  //withdraw button
+  var savingWithdrawButton = $("#savings .withdraw");
+
+  //if withdraw button clicked, updates total balance
+  savingWithdrawButton.on("click", function(){
+    var savingWithdrawValue = $("#savings .input");
+    savingWithdrawValue = parseInt(savingWithdrawValue.val());
+    savingAccountTotal = parseInt(savingAccountTotal);
+    if (savingWithdrawValue <= savingAccountTotal){
+      savingAccountTotal = savingAccountTotal - savingWithdrawValue;
+      savingAccountTotal = String(savingAccountTotal);
+      $("#savings .balance").html("$" + savingAccountTotal);
+    }
+    else if (savingWithdrawValue > savingAccountTotal){
+      checkingAccountTotal = parseInt(checkingAccountTotal);
+      var bothAccountsTotal = 0;
+      bothAccountsTotal = checkingAccountTotal + savingAccountTotal;
+      if (savingWithdrawValue <= bothAccountsTotal){
+        savingWithdrawValue = savingWithdrawValue - savingAccountTotal;
+        checkingAccountTotal = checkingAccountTotal - savingWithdrawValue;
+        checkingAccountTotal = String(checkingAccountTotal);
+        $("#checking .balance").html("$" + checkingAccountTotal);
+        savingAccountTotal = savingAccountTotal - savingAccountTotal;
+        savingAccountTotal = String(savingAccountTotal);
+        $("#savings .balance").html("$" + savingAccountTotal);
+      }
+      else {
+        alert("Sorry, there is not enough in your accounts to withdraw this much");
+      }
+
+    }
```

Comment:
this code looks very similar to the block of code above - can you think of a way to combine their functionality?